docs/terminal/sydney.md

```diff
@@ -340,63 +349,72 @@ Any aircraft that don't meet these criteria must be coordinated to SY TCU with a
     **SY ADC** -> **SY TCU**: "Next, ABC"  
     **SY TCU** -> **SY ADC**: "ABC, Heading 030"  
     **SY ADC** -> **SY TCU**: "Heading 030, ABC"  
-    **SY ADC** -> **ABC**: "ABC, Assigned heading right 030, Runway 34R, Cleared for Takeoff"  
-    **ABC** -> **SY ADC**: "Right heading 030, Runway 34R, Cleared for Takeoff, ABC"  
+
+    **SY ADC**: "ABC, Assigned heading right 030, Runway 34R, Cleared for Takeoff"  
+    **ABC**: "Right heading 030, Runway 34R, Cleared for Takeoff, ABC"  
     `AIP GEN 3.4`
 
 The SY TCU controller can suspend/resume Auto Release at any time, with the concurrence of **SY ADC**.
-### SY TCU / BK TWR
+### SY TCU / BK ADC
 #### Departures
 
-Aircraft departing YSBK in to SY TCU Class C will be coordinated from **BK TWR** at Taxi. Aircraft will need to be passed airways clearances to **BK TWR** at this point, to be relayed to the aircraft.
+Aircraft departing YSBK in to SY TCU Class C will be coordinated from **BK ADC** at Taxi. Aircraft will need to be passed airways clearances to **BK ADC** at this point, to be relayed to the aircraft.
 
 !!! example
-    **BK TWR** -> **SY TCU**: "Taxi, TFX12 for YMML via WOL"  
-    **SY TCU** -> **BK TWR**: "TFX12, BK, WOL, Flight Planned Route, A030, Squawk 3601"  
-    **BK TWR** -> **SY TCU**: "BK, WOL, Flight Planned Route, A030, Squawk 3601, TFX12"  
-    **BK TWR** Will then pass the airways clearance to TFX12
+    **BK ADC** -> **SY TCU**: "Taxi, TFX12 for YMML via WOL"  
+    **SY TCU** -> **BK ADC**: "TFX12, BK, WOL, Flight Planned Route, A030, Squawk 3601"  
+    **BK ADC** -> **SY TCU**: "BK, WOL, Flight Planned Route, A030, Squawk 3601, TFX12"  
 
-**BK TWR** will then give a "Next" call, where the SY TCU controller shall assign a heading (usually 290, for separation from YSSY).
+    **BK ADC** Will then pass the airways clearance to TFX12  
+    `AIP GEN 3.4`
+
+**BK ADC** will then give a "Next" call, where the SY TCU controller shall assign a heading (usually 290, for separation from YSSY).
 
 !!! example
-    **BK TWR** -> **SY TCU**: "Next, TFX12"  
-    **SY TCU** -> **BK TWR**: "TFX12, Right Heading 290, A030"  
-    **BK TWR** -> **SY TCU**: "Right Heading 290, A030, TFX12"  
-    **BK TWR** Will then clear the aircraft to takeoff with the assigned heading, and instruct them to contact SY TCU passing A020.
```

Comment:
Being pedantic here...

If no SID (VSD / IFR DEP), Right Heading 290 is NOT an assigned heading.
If BK8 SID, SID does not terminate on an assigned heading (as per plate), should be no heading used in coord not given to aircraft.
Additionally, if the heading is less than 5 degrees from Centreline, no turn direction need be given.